src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -10,53 +10,67 @@ public enum BusChain {
 	ROBORIO("rio"),
 	CANIVORE("CANivore");
 
+	private static final double MAX_CAN_UTILIZATION_PERCENT = 0.6;
+	private static final double MAX_RECEIVE_ERRORS = 0;
+	private static final double MAX_TRANSMIT_ERRORS = 0;
```

Comment:
do you mean the max errors received? if so change to MAX_ERRORS_RECEIVES

Reply:
its the max errors received in the "RECEIVE" action